utils/HtmlElems.ts

```diff
@@ -0,0 +1,183 @@
+import { CurrentButton, User } from "../src/UserApp"
+// Simple level 1 heading
+export const H1 = ( content: string ) => `<h1>${content}</h1>`
+
+// Div with size of 8 colums
+export const DivCol8OffSet = ( ...content: Array<string> ) => `<div class="col s8 offset-s2">
+    ${content.join( "" )}
+</div>`
+
+// Div with size of 4 colums
+export const DivCol4OffSet = ( ...content: Array<string> ) => `<div class="col s4 offset-s4">
+${content.join( "" )}
+</div>`
```

Comment:
These are the same 1 component. Instead of needlessly multiplying code, just make the offset an enum that can have states col4 and col8. Make this an argument of this component's function, and have 1 component where in its body you check the enum and put fitting variables into the string. e.g.
```ts
// Get ourselves a simple boolean we can later check to see if we are dealing with bigger offset
const bIsBigOffset = offset == DivOffset.col8;
const divClassStr = `col s${bIsBigOffset ? "s8" : "s4"} offset-s${bIsBigOffset ? "s2" : "s4"}`;

return `div class="${divClassStr}">${content.join( "" )}</div>`
```